server/test/dashboard.test.js

```diff
@@ -0,0 +1,100 @@
+import request from 'supertest';
+import { stub } from 'sinon';
+import server from '../app';
+import database from '../config/database';
+import Board from '../models/boardModel';
+import Dashboard from '../models/dashboardModel';
+
+server.close();
```

Comment:
Is this line intentional? If so,why?

Reply:
Yes, While running the test cases more than one times was giving an error that port already in use. to solve this I have added this line